fastlane_core/lib/fastlane_core/ui/interface.rb

```diff
@@ -135,8 +140,9 @@ def crash!(exception)
     #   stack trace
     # Basically this should be used when you actively catch the error
     # and want to show a nice error message to the user
-    def user_error!(error_message)
-      raise FastlaneError.new, error_message.to_s
+    def user_error!(error_message, options = {})
+      options[:show_github_issues] = true unless options.key?(:show_github_issues)
```

Comment:
You mean it should be
```ruby
error_message, options = { show_github_issues: true }
```
? 

I don't know if that works, I just figured it would be clear what we do by having a separate line. I'm more than happy to change it though